lnwallet/test/test_interface.go

```diff
@@ -1758,97 +1758,99 @@ func testPublishTransaction(r *rpctest.Harness,
 		tx3, tx3Spend *wire.MsgTx
 	)
 	t.Run("rbf_tests", func(t *testing.T) {
-		for _, rbf := range []bool{false, true} {
-			// Now we'll try to double spend an output with a
-			// different transaction. Create a new tx and publish
-			// it. This is the output we'll try to double spend.
-			tx3 = newTx(t, r, keyDesc.PubKey, alice, false)
-			err := alice.PublishTransaction(tx3, labels.External)
-			require.NoError(t, err)
-
-			// Mine the transaction.
-			err = mineAndAssert(r, tx3)
-			require.NoError(t, err)
+		// Starting with bitcoind v28.0 and later, mempool full RBF is
+		// turned on, so there's no way to _not_ signal RBF anymore.
+		const rbf = true
```

Comment:
i think this const can be removed, which allows us to remove some of the `if rbf` checks below.

Reply:
I left the const in, otherwise we'd pass a "naked" `true` into `txFromOutput`. But I removed the `if rbf {` blocks below.